.metamaskrc.dist

```diff
@@ -7,5 +7,8 @@ SWAPS_USE_DEV_APIS=
 COLLECTIBLES_V1=
 TOKEN_DETECTION_V2=
 
+; Set this to true to enable support for SIWE [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361)
+SIWE_VALIDATION=
```

Comment:
It looks like our other features prepend temporary feature flags with VX. Can we update this to something like `SIWE_V1`? I am reading the versioning as an indication to remove these flags later.